front/src/applications/stdcm/components/StdcmWarningBox.tsx

```diff
@@ -48,17 +48,28 @@ const StdcmWarningBox = ({
       >
         {t(`stdcmErrors.${errorType}`)}
       </p>
+      {errorType === StdcmConfigErrorTypes.MISSING_INFORMATIONS &&
+        errorDetails?.missingFields?.length && (
+          <div className="warning-missing-fields">
+            {errorDetails.missingFields.map((field) => (
+              <div key={field} className="warning-item">
+                <span className="bullet">&bull;</span>&nbsp;
+                {t(`stdcmErrors.missingFields.${field}`)}
+              </div>
+            ))}
+          </div>
```

Comment:
It seems like we don't need any class or `span` here
```suggestion
          <div>
            {errorDetails.missingFields.map((field) => (
              <div key={field}>&bull;&nbsp;{t(`stdcmErrors.missingFields.${field}`)}</div>
            ))}
          </div>
```

Reply:
done

Reply:
Looks like they are still there for the invalid informations

Reply:
Yes, i'm dealing with invalidFields now. It will be cleaner after the next push